etc/yum-cron-hourly.conf

```diff
@@ -20,8 +20,8 @@ download_updates = no
 # the update to be applied
 apply_updates = no
 
-# Days of week
-days_of_week="0123456"
+# You may set days_of_week to the numeric days of the week you want to run, where 0 is Sunday and 6 is Saturday. The default is to run every day.
```

Comment:
Awsome, one minor nitpick, though: please wrap the line to 80 chars :) (again, for consistency and ease of reading)